src/styles/blocks/header.scss

```diff
@@ -0,0 +1,18 @@
+@import "./utils/variables.scss";
+
+.header {
+  position: sticky;
```

Comment:
It doesn't work as you expect)) + there were no requirements about sticky header.

![image](https://user-images.githubusercontent.com/57747481/200617072-7c62d4a8-1b0b-4ead-a025-0bc283486598.png)